lib/galaxy/util/sanitize_html.py

```diff
@@ -112,17 +114,17 @@ def unknown_starttag(self, tag, attrs):
                 value = value.replace('>', '&gt;').replace('<', '&lt;').replace('"', '&quot;')
                 value = self.bare_ampersand.sub("&amp;", value)
                 # thanks to Kevin Marks for this breathtaking hack to deal with (valid) high-bit attribute values in UTF-8 feeds
-                if isinstance(value, unicode):
+                if isinstance(value, text_type):
                     try:
-                        value = unicode(value, self.encoding)
-                    except:
-                        value = unicode(value, 'iso-8859-1')
-                uattrs.append((unicode(key, self.encoding), value))
+                        value = unicodify(value, self.encoding)
+                    except Exception:
+                        value = unicodify(value, 'iso-8859-1')
```

Comment:
@mvdbeek Are you using feedparser from pip (i.e. 5.2.1) or from the develop branch on GitHub?

What was the problem with the `html` module? It should be fixable without a rename.

Anyway, I agree with @jmchilton, this is for another PR, can you create an issue for that and xref it in #1715?

For the time being, I suggest to remove all the unicodification here as it is now in https://github.com/kurtmckee/feedparser/blob/develop/feedparser/html.py .